popular-crates.txt

```diff
@@ -110,25 +101,22 @@ critcmp
 cross
 diener
 dinero-rs
-dors
 dotter
 dtn7
-duct
 du-dust
+duct
 dutree
 elf2tab
-emils-cargo-release
 emplace
-error_show
 exa
 fd-find
 feroxbuster
 ffsend
 flamegraph
 fw
 git-delta
-git_lab_cli
 git-state
+git_lab_cli
```

Comment:
I think that your version of sort might be ignoring - and _ characters, and only sorting alphabetic characters? I suppose we could use `sort --dictionary-order` if we want this?

Reply:
its a good point. can you fix that? i saw you added many more cool stuff on the way